styles.sass

```diff
@@ -0,0 +1,248 @@
+---
+---
+
+
+// variables 
+
+$dark-grey: #6F6B6B;
+$light-grey: #C6CECF;
+$light-blue: #8BCBFA;
+$medium-blue: #1097FA;
+$primary-text-bold: 700;
+$primary-text-italic: italic;
+
+
+// mixins
+
+@mixin nav-menu-links
+	padding: 0;
+	text-align: center;
+	list-style-type: none;
+
+@mixin accent-text
+	font-family: acier-bat-solid, sans-serif;
+	font-weight: 400;
+	font-style: normal;
+	color: $light-grey;
+
+@mixin primary-text
+	font-family: itc-galliard, serif;
+	font-weight: 400;
+	font-style: normal;
+	font-size: 14px;
+	line-height: 1.5;
+	color: $dark-grey;
+
+@mixin link-text
+	@include accent-text
+	font-size: 14px;
+	color: $light-blue;
+	text-decoration-line: none;
+	letter-spacing: 1px;
+	margin: 2px;
+
+
+@mixin primary-header
+	@include accent-text;
+	font-size: 24px;
+
+@mixin small-screen-section-spacing
```

Comment:
So I would probably only put mixins together that are re-used across the file. If a mixin is only re-used in multiple places that are very close together then I'd just put it near where its delcared